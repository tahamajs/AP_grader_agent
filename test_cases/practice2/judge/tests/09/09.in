enter 45
show_table_info 1
enter 10
reserve_table 30 1
enter 10
show_table_info 6
reserve_table 3 6
enter 10
exit 30
enter 10
show_table_info 6
reserve_table 7 6
enter 10
show_table_info 6
reserve_table 10
exit 10
reserve_table 10 6
show_table_info 6
reserve_table 41 6
show_table_info 6
exit 7
show_table_info 6
exit 3
exit 10
exit 41
show_table_info 1
reserve_table 1
reserve_table 3
reserve_table 4
reserve_table 26
reserve_table 24
reserve_table 15
reserve_table 7
reserve_table 8
show_table_info 1
reserve_table 28
reserve_table 27
reserve_table 29
reserve_table 30
reserve_table 44
show_table_info 2
reserve_table 23
reserve_table 38
reserve_table 10
reserve_table 9
reserve_table 6
reserve_table 2
reserve_table 5
reserve_table 12 5
switch 1
show_table_info 1
