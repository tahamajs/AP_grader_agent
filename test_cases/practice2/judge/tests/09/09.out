Table 1: 4 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Table ID: 1
People at the table:
Table remaining capacity: 4
Waiting queue length: 0
Table 1: 4 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Mike sits at table 1
Table 2: 4 0
Table 6: 3 0
Table 1: 3 0
Table 5: 3 0
Table 4: 2 0
Table 3: 2 0
Table ID: 6
People at the table:
Table remaining capacity: 3
Waiting queue length: 0
Parsa sits at table 6
Table 6: 2 0
Table 2: 4 0
Table 5: 3 0
Table 1: 3 0
Table 4: 2 0
Table 3: 2 0
Mike exits!
Table 2: 4 0
Table 6: 2 0
Table 1: 4 0
Table 5: 3 0
Table 4: 2 0
Table 3: 2 0
Table ID: 6
People at the table: Parsa
Table remaining capacity: 2
Waiting queue length: 0
Mahdi sits at table 6
Table 2: 4 0
Table 6: 1 0
Table 1: 4 0
Table 5: 3 0
Table 4: 2 0
Table 3: 2 0
Table ID: 6
People at the table: Mahdi, Parsa
Table remaining capacity: 1
Waiting queue length: 0
Hamid sits at table 2
Hamid exits!
Hamid sits at table 6
Table ID: 6
People at the table: Hamid, Mahdi, Parsa
Table remaining capacity: 0
Waiting queue length: 0
Melika enters the waiting queue of table 6
Table ID: 6
People at the table: Hamid, Mahdi, Parsa
Table remaining capacity: 0
Waiting queue length: 1
Mahdi exits!
Table ID: 6
People at the table: Hamid, Melika, Parsa
Table remaining capacity: 0
Waiting queue length: 0
Parsa exits!
Hamid exits!
Melika exits!
Table ID: 1
People at the table:
Table remaining capacity: 4
Waiting queue length: 0
Paria sits at table 1
Parsa sits at table 1
Mehrad sits at table 1
Walter sits at table 1
Sara enters the waiting queue of table 1
Erfan enters the waiting queue of table 1
Mahdi enters the waiting queue of table 1
Taha enters the waiting queue of table 1
Table ID: 1
People at the table: Mehrad, Paria, Parsa, Walter
Table remaining capacity: 0
Waiting queue length: 4
Hank sits at table 2
Jesse enters the waiting queue of table 1
Gustavo sits at table 2
Mike sits at table 2
Noah sits at table 2
Table ID: 2
People at the table: Gustavo, Hank, Mike, Noah
Table remaining capacity: 0
Waiting queue length: 0
Narges enters the waiting queue of table 1
Aryan enters the waiting queue of table 2
Hamid enters the waiting queue of table 1
Sadegh enters the waiting queue of table 1
Amir Akbar enters the waiting queue of table 1
Amir Hossein enters the waiting queue of table 1
Fereshte enters the waiting queue of table 1
Bob sits at table 5
Paria switches seats with Bob!
Table ID: 1
People at the table: Bob, Mehrad, Parsa, Walter
Table remaining capacity: 0
Waiting queue length: 11
